infra/base-images/base-builder-go/native_ossfuzz_coverage_runner.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mypackagebeingfuzzed
+
+import (
+	"io/ioutil"
+	"os"
+	"runtime/pprof"
+	"testing"
+	"github.com/AdamKorcz/go-118-fuzz-build/utils"
+)
+
+func TestFuzzCorpus(t *testing.T) {
```

Comment:
Maybe this should be called something like `GetCorpusCoverage`?

Reply:
Not a problem. Note that the name is the same as the existing go-fuzz coverage test:

- https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/ossfuzz_coverage_runner.go#L24
- https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder-go/ossfuzz_coverage_runner.go#L24

Reply:
Ah sorry, I'm OK with either then.